ons_alpha/jinja2/templates/base.html

```diff
@@ -481,4 +481,11 @@
 
 {% block scripts %}
     <script src="{{ static('js/main.js') }}"></script>
+    {% if load_highcharts_js %}
+        <script src="https://code.highcharts.com/highcharts.js"></script>
+        <script src="https://code.highcharts.com/modules/data.js"></script>
+        <script src="https://code.highcharts.com/modules/exporting.js"></script>
+        <script src="https://code.highcharts.com/modules/export-data.js"></script>
+        <script src="https://code.highcharts.com/modules/accessibility.js"></script>
```

Comment:
@helenb Could I get your help to fold this stuff into the static files pipeline? I tried, but couldn't get it to work!